debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -382,6 +391,23 @@ private static int validateMaxQueueSize(Configuration config, Field field, Field
         return count;
     }
 
+    private static int validateSkipOperation(Configuration config, Field field, ValidationOutput problems) {
+        String operations = config.getString(field);
+        for (String operation : operations.split(",")) {
```

Comment:
Let's also trim the values (removing leading/trailing whitespace), to be more lenient with input.

Reply:
Ofcourse, I'm sorry